package/src/Slider.tsx

```diff
@@ -221,9 +234,21 @@ const SliderComponent = (
       }
     : null;
 
+  const lowerLimit =
+    !!localProps.lowerLimit || localProps.lowerLimit === 0
+      ? localProps.lowerLimit
+      : LIMIT_MIN_VALUE;
+
+  const upperLimit =
+    !!localProps.upperLimit || localProps.upperLimit === 0
+      ? localProps.upperLimit
+      : LIMIT_MAX_VALUE;
+
```

Comment:
Setting defaults if the lower and upper limit props are not set.
The reason behind this is because for Android not setting them sends them as `0` to the native side which is a valid limit value.